content/products/dolthub/dolthub-api/database.md

```diff
@@ -103,6 +103,8 @@ headers = {
 }
 ```
 
+To upload the file, include two fields in the request body, `file` and `params`, the `file` should be type of `Blob`, and `params` should be a JSON object that includes `tableName`, `fileName`,  `fileType`, `importOp` and `primaryKeys` array.
+
```

Comment:
`tableName` is required if the upload is `fileType` `csv`, `psv`, or `xlsx`. `sql` and `any` types dont need a `tableName`